app/models/offer.rb

```diff
@@ -119,6 +119,15 @@ def scheduled?
     end
   end
 
+  def self.find_by_ggv_uuid(ggv_uuid)
+    begin
+      offer = GogovanOrder.find_by(ggv_uuid: ggv_uuid).delivery.offer
+    rescue Exception => e
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?<br>Useless assignment to variable - `e`.